packages/rsync.rb

```diff
@@ -28,6 +28,7 @@ class Rsync < Package
   depends_on 'lz4' # R
   depends_on 'openssl' # R
   depends_on 'popt' # R
+  depends_on 'py3_cmarkgfm' => :build
```

Comment:
Would this new dependency affect the existing binaries?

Reply:
It's only a build dependency to allow man pages to be generated correctly.

Reply:
Are man pages for the binaries generated correctly?